CONTRIBUTING.md

```diff
@@ -55,6 +55,23 @@ To run the same tests locally, find the corresponding pipeline definition (a
 to execute the tests (usually starting with `python -m pytest`) or it refers to
 a template file with the command.
 
+## Real-time communication
+
+For real time communication please join us on [Gitter](https://gitter.im/fairlearn/community?utm_source=share-link&utm_medium=link&utm_campaign=share-link). All are welcome!
```

Comment:
Is the plan to shut down the existing 'fairlearn' org on Teams in favour of this?

Reply:
No, at least not yet. We want to get familiar with Gitter first, and see how it goes there. I'd hate to disrupt our ongoing collaborations, so you can rest assured to have it until at least end of June :-)
Other than those collaborations there's pretty much no usage of that Teams team.

Reply:
Any concerted effort should still be coordinated on Teams for the time being. We can re-consider the platform as we grow.

Reply:
I think that "Real-time communication" and "Meetings" should go into `README.md` under a new section called Community. That section should also mention that any problems should be ideally raised as issues (and be possibly merging in some other sections from `README.md`).

Reply:
One thing to note is that the "we'll move to platform X when the community reaches Y members" can be kind of a chicken and egg problem. Certain decisions may be hampering the growth of the community.

Reply:
@MiroDudik community is a very broad term. I'm sure users won't show up for "meetings". If I have a usage question (one form of issue) with scikit-learn I go to StackOverflow, or perhaps Gitter, but I won't wait for their developer call to explain my issue. Issue could also mean feature request or bug, which would belong on GitHub. That's why there is another PR to have bug/feature request templates for issues, which clearly indicate that usage questions belong on StackOverflow or Gitter. 

Real-time Communication and meetings are for contributors, not necessarily users (not that we wouldn't answer their questions if they choose the wrong path). Putting it into the README provides the opposite signal.

I also agree with @adrinjalali - the longer we wait with trying Gitter or other platforms the harder it will be. Teams is not a tool that open source projects commonly use as of today, so that represents a barrier to entry. We can't really afford to build up any of those. I'm not saying we should delete the existing Teams team, but let's make an effort to use Gitter (with the exception of the existing collaborations which are temporary anyway). If we notice severe issues that make us abandon that tool, no problem, let's move on elsewhere (or back to Teams).

At least that's my $0.02 :-)

Reply:
The current versions of CONTRIBUTING.md and README.md look good to me. Closing the thread. Let's see how things go on Gitter!